styleguide/STYLE.md

```diff
@@ -0,0 +1,224 @@
+| Signup/login              | When referring to a page that enables signing up or to the signup process                        |
+| API, SSO, SAML            | Always capitalized |
+| GitHub                    | Always capitalized, with an uppercase H in the middle                                            |
+| Two-factor authentication | In a sentence two-factor authentication, in a title Two-Factor Authentication, in short form 2FA |
+| Single sign-on            | In a sentence single sign-on, in a title Single Sign-On, in short form SSO                       |  
+
+### Common trip-ups
+Linters cannot do all of the work for you, so please pay attention to the following cases:
+
+* **Their/they’re/there, your/you’re** - [check](https://www.dictionary.com/e/their-there-theyre/ ) if you’ve got the right one for your situation!  
+* **Affect/effect** - affect is a verb, effect is a noun. When you affect something, you’re impacting or changing the thing. When you have an effect, it’s the outcome or result of a change. 
+* Be mindful of **hyphens**! Hyphens for compound adjectives, no hyphens in verbs: 'end-user documentation' vs. 'for the end user'.
+
+### Headings  
+No multi-sentence headings or full stops at the end of a sentence in page or sentence headings. 
+Be consistent about heading levels - H1, H2, H3 - no jumping from H1 to H3 or H2 to H4.
+
+## Code and filenames  
+This section deals with adding and properly formatting code in the documentation + naming files, pages, and their derivative URLs.  
+
+### Code formatting  
+We use the GitHub flavor of markdown for [formatting code](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code). 
+
+For code or file names as a part of a sentence, use "\`\" before and after the word(s) that need(s) to be marked as code: Each command step can run either a shell command like `npm install`, or an executable file or script like `build.sh`. 
+In markdown this sentence looks like this: " Each command step can run either a shell command like \`npm install\`, or an executable file or script like \`build.sh\`. "
+
+Do not use code fragments in page headings or section headings.
+
+### Code blocks  
+A code example longer than a couple of words that isn’t part of a sentence/a multi-line code sample needs to be formatted as a code block according to the [GitHub markdown flavor](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code).
+To add a code block, indent it using four (4) spaces or use 3 backticks (\`\`\`) before and after the code block.
+
+```
+Hello, world!
+```
+
+To add a filename to a codeblock, immediately after the block use `{: codeblock-file="filename.extension"}` (for example, `{: codeblock-file="pipeline.yml"}`).   
+
+To add syntax highlighting, you can use [Rouge](http://rouge.jneen.net/), for example:  
+```
+```bash
+#!/bin/sh
+echo "Hello world"```
+```  
+turns into 
+```bash
+#!/bin/sh
+echo "Hello world"
+```  
```

Comment:
```suggestion
    ```bash
    #!/bin/sh
    echo "Hello world"
    ```  
```

this is a little simpler, and should do the trick

Reply:
Up to you, and apologies for reviving a completed PR. ;-)